src/index.js

```diff
@@ -77,6 +79,10 @@ module.exports = function(babel) {
         // Only walk specifiers if this is a module we have a mapping for
         if (mapping) {
 
+          if (mapping.deprecated) {
+            notifyDeprecation(`Module ${importPath} is deprecated.`, notifier);
```

Comment:
We should flesh this out a bit more to include what file was being transpiled and what the replacement is (if any).  Otherwise this isn't very actionable.

@Serabe - I'm going to disable the deprecation system for now, so that we can release (and keep parity). Can you work on this in a follow up PR?

Reply:
I created https://github.com/ember-cli/babel-plugin-ember-modules-api-polyfill/issues/28 to track that.